tests/integration/command-functions.test.ts

```diff
@@ -113,6 +117,30 @@ describe('Process creation and deletion, listing', () => {
         version: bumpedVersion,
         status: 'Disabled',
       })
+
+      /*
```

Comment:
was not aware that this will get pushed over this branch. could you please remove comments. if this will be included as part of another PR?